README.md

```diff
@@ -1,13 +1,26 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
+For this project I have used Mongodb to store and retrieve data from a RESTful API. The data is fetched from a json file and populated to the database.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The API has different routes for tracks and artist.
 
-## The problem
+## Documentation
+
+To find data in the API
+
+List all tracks:
+https://api-top-music.herokuapp.com/tracks
+
+List all track with query, example:
+https://api-top-music.herokuapp.com/tracks?genre=pop
```

Comment:
Unfortunately, I get: Application error
An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command
Heroku logs --tail
😭

Reply:
I updated the config vars so now it's working.